src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/PniRiskEngine.kt

```diff
@@ -55,21 +55,17 @@ class PniRiskEngine {
 
   private fun isOspDcHigh(individualRiskScores: IndividualRiskScores, gender: String): Boolean {
     return gender.equals("Male", ignoreCase = true) &&
-      (
-        individualRiskScores.ospDc?.equals("HIGH", ignoreCase = true) == true ||
-          individualRiskScores.ospDc?.equals("VERY_HIGH", ignoreCase = true) == true
-        )
+      (individualRiskScores.ospDc?.contains("HIGH", ignoreCase = true) == true)
   }
 
   private fun isOspIicHigh(individualRiskScores: IndividualRiskScores, gender: String): Boolean {
     return gender.equals("Male", ignoreCase = true) &&
-      (individualRiskScores.ospIic?.equals("HIGH", ignoreCase = true) == true) ||
-      (individualRiskScores.ospIic?.equals("VERY_HIGH", ignoreCase = true) == true)
+      (individualRiskScores.ospIic?.contains("HIGH", ignoreCase = true) == true)
```

Comment:
Minor: I know it doesn't matter with the ignore case, but can you change the "HIGH" to "High" just to keep it consistent?